Makefile

```diff
@@ -20,11 +20,13 @@ SF_ECLINT_FILES_IGNORE := \
 	-e "^rules/no-for\.js$$" \
 	-e "^rules/object-curly-newline\.js$$" \
 	-e "^rules/order-imports\.js$$" \
+	-e "^rules/sort-object-pattern\.js$$" \
```

Comment:
Kind of don't like this practice. Anything smarter we can do? Maybe just put these rules in a folder `from-eslint` or `eslint-official`?

Reply:
we can make a deal to discuss this after 2 more imported rules ;) why 2? because i somehow think it's not going to happen :) 1 more imported rule, maybe maybe maybe, 2 more, never :p 

what would be nice was editorconfig supporting inlined config `/* editorconfig-disable */`